css.md

```diff
@@ -0,0 +1,34 @@
+# CSS
+
+**Prerequisite knowledge**: basic html elements, imagination
```

Comment:
What do you mean by "imagination"? Is it something you can learn about? Do we plan to create an article about it?

The initial idea was to list only documents that are describing given topic